app/helpers/current_user.rb

```diff
@@ -0,0 +1,5 @@
+helpers do
+  def current_user
+    @current_user ||= User.get(session['user_id']) if session['user_id']
```

Comment:
I think what you're trying to do here is protect from current_user returning nil values. (is that right?) Unfortunately this method will still return nil when session['user_id'] is empty. Notably your tests still pass when the conditional is deleted.